src/main/java/com/incentives/piggyback/user/util/constants/Constant.java

```diff
@@ -7,4 +7,6 @@
     String USER_CREATED_EVENT = "User Events Created";
     String USER_UPDATED_EVENT = "User Events Updated";
     String USER_DEACTIVATED_EVENT = "User Events Deactivated";
+
+    String USER_TYPE_FB = "FACEBOOK_USER";
```

Comment:
This should be coming from the Roles.java enum rather than as a string

Reply:
its user type not role type , also in getAllUserRole() we will get this also then.

Reply:
Ok. Then what is the default role?